source/index.txt

```diff
@@ -32,6 +32,11 @@ installing the {+driver-short+}, see
 :ref:`Download and Install <node-quick-start-download-and-install>` in the
 Quick Start guide.
 
+You can connect using the Node.js driver for
+deployments hosted in the following environments:
```

Comment:
I: I don't love that the Style Guide makes this an absolute, but [it prohibits standalone "using".](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#u:~:text=word%2C%20username.-,using,-by%20using%20with) I think something like this is clearer, too:

```suggestion
You can use the Node.js driver to connect to
deployments hosted in the following environments:
```